ext/civiimport/Civi/Api4/Event/Subscriber/ImportSubscriber.php

```diff
@@ -176,6 +176,10 @@ public function onApiAuthorize(AuthorizeEvent $event): void {
       if (!UserJob::get(TRUE)->addWhere('id', '=', $userJobID)->selectRowCount()->execute()->count()) {
         throw new UnauthorizedException('Import access not permitted');
       }
+      if ($event->getActionName() === 'get') {
```

Comment:
`UserJob::get` succeeds because the user has permission for that. But then then don't have permission for eg. `Import_208.get` - feels like the intention was that if they could access UserJob.get with the `$userJobID` they could also access the equivalent `Import_X.get` API?

Reply:
Yeah that sounds right - maybe our users have `administer CiviCRM` & hence they are not hitting this. I guess it might be under-tested without that permission

Reply:
Thanks @eileenmcnaughton